src/repositories/user-repository.ts

```diff
@@ -26,9 +29,8 @@ export class UserRepository {
         name: 'asc',
       },
       take: limit,
+      skip: skip,
```

Comment:
```suggestion
      skip,
```
curiosidade legal do js, se a chave tem o mesmo nome do valor, pode omitir